### PR TITLE
win_nssm - Implement additional parameters

### LINKED ISSUE #62620: win_nssm Is setting some defaults that should have an option to overwrite
##### SUMMARY

The win_nssm module has a few defaults set now that I think should allow for customization. For example I'd like to be able to set the following:
`AppRotateBytes 1000000` 
`AppRotateOnline 1`
`AppStopMethodConsole 15000`
`AppStopMethodSkip 6`. 

I had been setting them via win_shell command, but now the module and shell commands are battling one another. 

https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/windows/win_nssm.ps1#L449
https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/windows/win_nssm.ps1#L439

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME

win_nssm.ps1

### PR DESCRIPTION
**Pull request by ShachafGoldstein**
##### SUMMARY
Fixes: #62620

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
win_nssm.ps1

##### ADDITIONAL INFORMATION

### CODE REVIEW

File: lib/ansible/modules/windows/win_nssm.ps1 [hunk 1]
```diff
@@ -38,6 +38,12 @@ $stderrFile = Get-AnsibleParam -obj $params -name "stderr_file" -type "path"
 
 $executable = Get-AnsibleParam -obj $params -name "executable" -type "path" -default "nssm.exe"
 
+$executable = Get-AnsibleParam -obj $params -name "executable" -type "path" -default "nssm.exe"
+$AppRotateBytes = Get-AnsibleParam -obj $params -name "AppRotateBytes" -type "int" -default 104858
```
Comment (jborean93) [inline lib/ansible/modules/windows/win_nssm.ps1]: The parameters should follow the `snake_case` format to match other Ansible module options.

**Review by jborean93 — CHANGES_REQUESTED** [review]
Looks good, just need to change the format of the module option names to snake_case

File: lib/ansible/modules/windows/win_nssm.ps1 (hunk 1 above)
Comment (ShachafGoldstein) [reply lib/ansible/modules/windows/win_nssm.ps1]: Other app related params are camel, should I change them as well and add aliases?

Comment (jborean93) [reply lib/ansible/modules/windows/win_nssm.ps1]: Yep all module options should be in snake_case

**Comment by ShachafGoldstein** [comment]
Fixed to snake and a missing if

File: lib/ansible/modules/windows/win_nssm.ps1 [hunk 2]
```diff
@@ -38,6 +38,12 @@ $stderrFile = Get-AnsibleParam -obj $params -name "stderr_file" -type "path"
 
 $executable = Get-AnsibleParam -obj $params -name "executable" -type "path" -default "nssm.exe"
 
+$executable = Get-AnsibleParam -obj $params -name "executable" -type "path" -default "nssm.exe"
```
Comment (jborean93) [inline lib/ansible/modules/windows/win_nssm.ps1]: This is a double up of the above.

**Review by jborean93 — APPROVED** [review]
Just had to remove a duplicate line and add the defaults to the options but it looks good to me.

**Pull request merged**